scripts/participateInAuction.js

```diff
@@ -1,19 +1,25 @@
 /* eslint-disable no-console */
 const Coordinator = artifacts.require("Coordinator")
+const { Gastimator } = require("./gas_station")
+
+const GasStation = new Gastimator()
+
+// TODO - make this part of configuration
+const url = "https://safe-relay.rinkeby.gnosis.pm/api/v1/gas-station/"
 
 module.exports = async (callback) => {
   try {
     const coordinator = await Coordinator.deployed()
     console.log("Coordinator deployed at %s", coordinator.address)
     if (await coordinator.canParticipate()) {
-      await coordinator.participateInAuction()
+      const gasPriceEstimates = await GasStation.estimateGas(url)
+      // Send transaction with fast gas price estimate
+      await coordinator.participateInAuction({ "gasPrice": gasPriceEstimates["fast"] })
```

Comment:
nit: gasPriceEstimates.fast should work?

Reply:
Thats awesome, I'll give it a try.

Reply:
Done.